src/components/Landing/EventCard/EventCardModal/EventCardModal.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import "./EventCardModal.scss";
+import PropTypes from "prop-types";
+import { DELETE_DRAFT } from "../../../../constants/draftConstants";
+
+const EventCardModal = ({ closeModal, id, onClose, handleClick }) => {
+  const [reason, setReason] = useState("");
+  const confirm = () => {
+    handleClick(id, reason);
+    setReason("");
+    onClose(!closeModal);
+  };
+  return (
+    <div className="wrapper">
+      <div className="container">
+        <div className="titleText">
+          <h4 className="title">{DELETE_DRAFT.MESSAGE}</h4>
+        </div>
```

Comment:
extra wrapper?

Reply:
+

Reply:
why did you close this comment? it is valid

Reply:
title classname has styles in general style file and it didnt work, I will try again